packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -207,9 +245,28 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
     };
   }
 
+  private _isOffsetHeightDifferent(): boolean {
+    const { scrollablePane } = this.context;
+    if (!scrollablePane) {
+      return false;
+    }
+    const usePlaceholderForSticky = scrollablePane.usePlaceholderForSticky();
+    const usePlaceholderForStickyContentTop = this.canStickyTop && usePlaceholderForSticky;
+    const usePlaceholderForStickyContentBottom = this.canStickyBottom && usePlaceholderForSticky;
+    return (
+      (usePlaceholderForStickyContentTop && _isOffsetHeightDifferent(this._nonStickyContent, this._stickyContentTop)) ||
+      (usePlaceholderForStickyContentBottom && _isOffsetHeightDifferent(this._nonStickyContent, this._stickyContentBottom)) ||
+      ((usePlaceholderForStickyContentBottom || usePlaceholderForStickyContentTop) &&
+        _isOffsetHeightDifferent(this._nonStickyContent, this._placeHolder))
+    );
+  }
+
   private _getNonStickyPlaceholderHeightAndWidth(): React.CSSProperties {
     const { isStickyTop, isStickyBottom } = this.state;
-    if (isStickyTop || isStickyBottom) {
+    const usePlaceholderForSticky = this.context.scrollablePane.usePlaceholderForSticky();
```

Comment:
> const usePlaceholderForSticky = this.context.scrollablePane.usePlaceholderForSticky() [](start = 3, length = 86)

NIT: Should we create a function for this since it used in multiple places ?